pkg/bosh/manifest/instance_group.go

```diff
@@ -96,10 +96,16 @@ type JobDir struct {
 	TmpfsSize string `yaml:"tmpfs_size,omitempty"`
 }
 
+// Agent from BOSH deployment manifest
```

Comment:
How about also explaining what these settings are for, something like:
```suggestion
// AgentSettings from BOSH deployment manifest. These annotations and labels are added to ....
```